testApi/src/main/scala/africa/absa/testing/scapi/rest/response/Response.scala

```diff
@@ -16,7 +16,7 @@
 
 package africa.absa.testing.scapi.rest.response
 
-import africa.absa.testing.scapi.json.ResponseAction
+import africa.absa.testing.scapi.json.{Environment, ResponseAction}
```

Comment:
I think `Environment` import is not needed here

Reply:
Fixed in last commit.